blockchain/treasury.go

```diff
@@ -1089,7 +1089,7 @@ func (b *BlockChain) checkTSpendHasVotes(prevNode *blockNode, tspend *dcrutil.Tx
 			maxRemainingVotes)
 	}
 
-	trsyLog.Infof("TSpend %v passed with: yes %v no %v quorum %v "+
+	trsyLog.Debugf("TSpend %v passed with: yes %v no %v quorum %v "+
```

Comment:
Since this is being updated, what do you think about making it more consistent with the rest of the logging as follows?  It's a minor thing, but the inconsistency bugged me when I saw it.

```suggestion
	trsyLog.Debugf("Treasury spend %v passed (yes %v, no %v, quorum %v, "+
		"required %v)", tspend.Hash(), t.yes, t.no, quorum, requiredVotes)
```

Reply:
Good call, I prefer that as well.  Updated.